airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/JdbcBufferedConsumerFactory.kt

```diff
@@ -152,7 +170,9 @@ object JdbcBufferedConsumerFactory {
     ): OnStartFunction {
         return OnStartFunction {
```

Comment:
I missed this on my initial review - where do we handle the sync startup logic around checking for temp raw table existence / generation id?

Reply:
the createWriteConfig function does that by checking the table's data